website/source/docs/job-specification/group.html.md

```diff
@@ -65,6 +65,13 @@ job "docs" {
   all tasks in this group. If omitted, a default policy exists for each job
   type, which can be found in the [restart stanza documentation][restart].
 
+- `shutdown_delay` `(string: "0s")` - Specifies the duration to wait when
+  killing a groups tasks between removing it from Consul and sending each task
```

Comment:
I think this could be clearer? Here's an attempt:

```suggestion
  stopping a group's tasks. The delay occurs between Consul deregistration and sending each task
```